rca/home/migrations/0003_homepage_blocks_and_fields.py

```diff
@@ -0,0 +1,293 @@
+# Generated by Django 2.2.4 on 2019-08-22 23:16
+
+import django.db.models.deletion
+import modelcluster.fields
+import wagtail.core.blocks
+import wagtail.core.fields
+import wagtail.images.blocks
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("images", "0002_customimage_file_hash"),
+        ("home", "0002_create_homepage"),
+    ]
+
+    operations = [
+        migrations.RemoveField(model_name="homepage", name="call_to_action"),
+        migrations.AddField(
+            model_name="homepage",
+            name="hero_colour_option",
+            field=models.CharField(
+                choices=[
+                    ("1", "Light text on a dark image"),
+                    ("2", "Dark text on a light image"),
```

Comment:
@tm-kn I attempted changing this to an integer field but it caused a database error `django.db.utils.DataError: invalid input syntax for integer: ""`

Reply:
Did you change "1" -> 1?

Reply:
yep, I swapped the string out for the int and changed the field e.g
```
hero_colour_option = models.PositiveSmallIntegerField(
        choices=(
            (1 "Light text on a dark image"),
            (2, "Dark text on a light image"),
        ),
    )
```

Reply:
Did you change the `default` as well? It seems to be `""`. If integer field can be empty, it should be `null=True`.

Reply:
I've implemented this